test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -803,6 +886,23 @@ Describe "Invoke-WebRequest tests" -Tags "Feature" {
         $response.Content.UserAgent | Should Match $Pattern
     }
 
+    It "Verifies Invoke-WebRequest Certificate Authentication Fails without -Certificate"{
+        $command = 'Invoke-WebRequest https://prod.idrix.eu/secure/'
```

Comment:
@markekraus Have you looked to see if there is some Linux (so it would work on both platforms) docker container that would work?  Perhaps ASP.NET core in Linux should be familiar to most people using this platform.